### PR TITLE
Support non-btc pairs for btcbox

### PR DESCRIPTION
**Pull request by xyc0562**
This commit enables btcbox adapter to work with non-btc pairs

### CODE REVIEW

File: js/btcbox.js [hunk 1]
```diff
@@ -190,12 +190,17 @@ module.exports = class btcbox extends Exchange {
 
     async cancelOrder (id, symbol = undefined, params = {}) {
         await this.loadMarkets ();
+        if (!symbol) {
+            symbol = 'BTC/JPY';
```
[the remark this answers is not in the document]
Comment (xyc0562) [reply js/btcbox.js]: Hi!

I have made changes as requested and made another commit 😄

**Comment by xyc0562** [comment]
Please let me know if current commits are OK? I can squash the commits as well if needed.

**Comment by xyc0562** [comment]
@kroitor can we merge this request if everything is fine 😄 ?

**Comment by kroitor** [comment]
@xyc0562 yep, will do that shortly! Thx so much for your participation! )

**Pull request merged**